services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+from typing import Optional
+
+
+User = get_user_model()
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None

```

Comment:
The use of `Optional[str]` is appropriate here to indicate that these fields can be `None`. However, consider using default values directly in the `create_user` method to simplify the logic.